packages/vuetify/src/composables/outlined.ts

```diff
@@ -0,0 +1,31 @@
+// Utilities
+import { computed } from 'vue'
+import propsFactory from '@/util/propsFactory'
+
+// Types
+export interface OutlinedProps {
+  outlined?: boolean | string
+}
+
+// Composables
+export const makeOutlinedProps = propsFactory({
+  outlined: [Boolean, String],
+})
+
+export function useOutlined (props: OutlinedProps, target: string) {
```

Comment:
`target` is interpolated into class, so it needs a union type of values, not `string`

Reply:
this has been removed.